manifests/core-install/kustomization.yaml

```diff
@@ -13,3 +13,18 @@ images:
 - name: quay.io/argoproj/argocd
   newName: quay.io/argoproj/argocd
   newTag: latest
+
```

Comment:
I had to duplicate this replacement, the core install manifest generation does not use the entire base.